### PR TITLE
Upper-even mutations

### PR DESCRIPTION
**Pull request by alxndralsn**
5 mutations and 1 solution for upper-even

### CODE REVIEW

File: upper-even/mutation_1.py
Comment (adr-diana) [inline upper-even/mutation_1.py:1]: you need to explain why this mutation fails the test cases

File: upper-even/solution.py
Comment (adr-diana) [inline upper-even/solution.py:1]: all good

File: upper-even/mutation_2.py
Comment (adr-diana) [inline upper-even/mutation_2.py:1]: all good

File: upper-even/mutation_1.py
Comment (adr-diana) [reply upper-even/mutation_1.py:1]: also put Mutation 1: to keep it uniform (like in the other files)

File: upper-even/mutation-1.py
Comment (adr-diana) [inline upper-even/mutation-1.py:1]: all good, but you only did two mutations? We need to do minimum 3

**Review by adr-diana — APPROVED** [review]

File: upper-even/mutation_3.py
Comment (adr-diana) [inline upper-even/mutation_3.py:1]: you need to explain why the mutations make the test cases fail

File: upper-even/mutation_4.py
Comment (adr-diana) [inline upper-even/mutation_4.py:1]: all good

File: upper-even/mutation_5.py
Comment (adr-diana) [inline upper-even/mutation_5.py:1]: all good

**Comment by emaj9** [comment]
great teamwork!

**Pull request merged**